src/DataExplorer/actionHelper.js

```diff
@@ -218,7 +208,7 @@ export const getManifestEntryCount = async (
     arrangerConfig.manifestMapping.resourceIndexType,
     [{
       name: arrangerConfig.manifestMapping.referenceIdFieldInResourceIndex,
-      values:selectedTableRows
+      values: selectedTableRows,
```

Comment:
Thanks a lot for fixing the lint and moving unused arguments! could also remove `graphqlIdField` from this `getManifestEntryCount` function since it is not used.

Reply:
good catch! removed in https://github.com/uc-cdis/data-portal/pull/538/commits/9ce4f53422a1f0611b6fa5afd0b11a1dc76c8a2f